CHANGELOG.md

```diff
@@ -899,6 +899,7 @@ All notable changes to this project will be documented in this file.
 [`useless_let_if_seq`]: https://rust-lang.github.io/rust-clippy/master/index.html#useless_let_if_seq
 [`useless_transmute`]: https://rust-lang.github.io/rust-clippy/master/index.html#useless_transmute
 [`useless_vec`]: https://rust-lang.github.io/rust-clippy/master/index.html#useless_vec
+[`vec_box_sized`]: https://rust-lang.github.io/rust-clippy/master/index.html#vec_box_sized
```

Comment:
let's not call it `_sized`, the sized thing is for filtering false positives but is not a necessary keyword for the lint itself. `vec_box`?

Reply:
:+1: